controller/iteration_blackbox_test.go

```diff
@@ -612,21 +611,22 @@ func (rest *TestIterationREST) createIterations() (*app.IterationSingle, *accoun
 	return created, owner
 }
 
-// TestIterationActivedByUser tests iteration should always be active when user sets it to active
+// TestIterationActivedByUser tests:
+// 1. Iteration should be active when it is in timeframe
+// 2. Iteration should not be active when it is outside the timeframe
+// 3. Iteration should always be active when user sets it to active
```

Comment:
These tests can be perfectly mapped into sub tests. Please make use of them to increase readability and maintainability.

Reply:
@kwk worked on this here 7d05c54